### PR TITLE
corrections of the profile page and link the page to the edit profile…

### PR DESCRIPTION
**Pull request by Fabre-92**
… page

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -13,53 +13,79 @@
...
+                  <div class="d-flex justify-content-left">
+                    <i class="far fa-calendar-alt mt-1 text-primary"></i>
+                    <div class="ml-3">
+                      <p class="mb-0"><small>from: </small> <%= booking.start_date.strftime("%A, %b %d") %> </p>
+                      <p> <small>to:</small> <%= booking.end_date.strftime("%A, %b %d") %> </p>
+                    </div>
+                  </div>
+                  <div class="d-flex justify-content-between align-items-center">
+                    <p class="mt-1 mb-0"> <i class="fas fa-toolbox mr-2"></i> <strong> <%= booking.tool.name %> </strong> </p>
+                    <%= link_to edit_user_registration_path do %>
```
Comment (Fabre-92) [inline app/views/users/show.html.erb:85]: change the link to the correct link

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -69,28 +95,34 @@
...
+           <div class="user-info">
               <h5 class="card-title">Passionate gardener living in Beirut</h5>
-              <p class="card-text"> Contact information:</p>
-              <ul>
-                <li> EMAIL: <%= @user.email %></li>
-                <li> NAME: <%= @user.first_name %> <%= @user.last_name %></li>
+              <p class="card-text"> <i class="fas fa-tools"></i> User info:</p>
+              <ul class="ul_decoration">
+                <li> <i class="fas fa-upload small ml-0.5 mr-2 text-secondary"></i><strong><%= @tools.count %></strong>  <small>Tools uploaded</small></li>
+                <li> <i class="fab fa-get-pocket small mr-2 text-secondary"></i> <%=  %></li>
```
Comment (Fabre-92) [inline app/views/users/show.html.erb:123]: add the ruby here to do the logic

File: app/views/users/show.html.erb (hunk 1 above)
Comment (helenagagern) [reply app/views/users/show.html.erb:85]: link_to edit_tool_path(@tool)

File: app/views/users/show.html.erb (hunk 2 above)
Comment (helenagagern) [reply app/views/users/show.html.erb:123]: @bookings.count

**Pull request merged**